### PR TITLE
change all constants to UpperCase, be careful when you use them next time.

### PR DESCRIPTION
**Pull request by amazingandyyy**
try to kill issue https://github.com/bodhiproject/bodhi-ui/issues/707

and also the merge the RouterPath and AppLocation

### CODE REVIEW

**Comment by amazingandyyy** [comment]
big change to `BackButton` and `NavLink`, others just merge the RouterPath and AppLocation.

File: src/scenes/Activities/index.js [hunk 1]
```diff
@@ -60,18 +60,18 @@ export default class Activities extends Component {
...
-      [set]: AppLocation.resultSet,
-      [finalize]: AppLocation.finalize,
-      [withdraw]: AppLocation.withdraw,
-      [activityHistory]: AppLocation.activityHistory,
+      [SET]: AppLocation.SET,
+      [FINALIZE]: AppLocation.FINALIZE,
+      [WITHDRAW]: AppLocation.WITHDRAW,
+      [ACTIVITY_HISTORY]: AppLocation.ACTIVITY_HISTORY,
     };
     const appLocation = locations[this.props.match.path];
```
Comment (alex-cory) [inline src/scenes/Activities/index.js]: I believe we can actually do `const location = _.invert(AppLocation)[this.props.match.path]`

Comment (alex-cory) [reply src/scenes/Activities/index.js]: Then we can remove `const locations = {` object

File: src/scenes/Activities/index.js [hunk 2]
```diff
@@ -83,19 +83,19 @@ export default class Activities extends Component {
     let label;
```
Comment (alex-cory) [inline src/scenes/Activities/index.js]:
```js
const eventStatus = _.invert(EventStatus)[eventStatusIndex];
const label = {
  SET: '...',
  FINALIZE: '...',
  WITHDRAW: '...',
}[eventStatus]
const count = actionableItemCount[eventStatus]
```

File: src/constants.js [hunk 3]
```diff
@@ -34,59 +20,57 @@ module.exports = {
...
+    WITHDRAW: 'WITHDRAW',
+    WITHDRAW_ESCROW: 'WITHDRAWESCROW',
+    TRANSFER: 'TRANSFER',
+    RESET_APPROVE: 'RESETAPPROVE',
   },
 
   TransactionStatus: {
-    Pending: 'PENDING',
+    PENDING: 'PENDING',
     Success: 'SUCCESS',
```
Comment (alex-cory) [inline src/constants.js]: is this not used either?

File: src/scenes/Activities/index.js (hunk 1 above)
Comment (amazingandyyy) [reply src/scenes/Activities/index.js]: updated

File: src/constants.js (hunk 3 above)
Comment (amazingandyyy) [reply src/constants.js]: we are using it

**Comment by amazingandyyy** [comment]
@alex-cory  can you approve it thanks

**Pull request merged**